docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,96 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
```

Comment:
In the beginning of what? When you first create the cluster? Or something else?

Reply:
When the thing using the persistent volume was first created...not sure how best to phrase that. Suggestions welcome!

Reply:
I would rephrase to "You may need to resize your persistent volumes as your needs change."